src/wp-admin/includes/class-wp-list-table.php

```diff
@@ -388,7 +388,16 @@ public function search_box( $text, $input_id ) {
 		$input_id = $input_id . '-search-input';
 
 		if ( ! empty( $_REQUEST['orderby'] ) ) {
-			echo '<input type="hidden" name="orderby" value="' . esc_attr( $_REQUEST['orderby'] ) . '" />';
+			if ( is_array( $_REQUEST['orderby'] ) ) {
+				foreach ( $_REQUEST['orderby'] as $key => $value ) {
+					// Because arrays in query parameters are not standardized, the orderby parameter can be an associative or a non-associative array.
+					// So in the latter case, we have to make sure we're passing a string to the escape function later on.
```

Comment:
A couple of nit-picks:
* use coding standards for multi-line comments
* avoid the use of the self referential "we". WordPress tries to avoid it as much as possible.

```suggestion
					/*
					 * Orderby can be either an associative array or non-associative array.
					 * In the latter case, this makes sure the key is a string before calling esc_attr().
					 */
```

Reply:
Done :)